packages/gatsby/src/redux/actions/public.ts

```diff
@@ -41,17 +75,17 @@ const {
   getInProcessJobPromise,
 } = require(`../../utils/jobs-manager`)
 
-const actions = {}
+export const actions: Record<string, any> = {}
```

Comment:
can we change this to just export each function instead of this? I know this was the old way, but it didn't seem useful

Reply:
Done in 27b2919